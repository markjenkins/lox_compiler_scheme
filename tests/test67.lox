// from craftinginterpreters/test/precedence.lox

// * has higher precedence than +.
print 2 + 3 * 4; // expect: 14
