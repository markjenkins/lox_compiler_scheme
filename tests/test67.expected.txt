OP_CONSTANT 2
OP_CONSTANT 3
OP_CONSTANT 4
OP_MULTIPLY
OP_ADD
OP_PRINT
OP_RETURN
